app/assets/stylesheets/refinery/find_out.scss

```diff
@@ -17,31 +17,110 @@
   .banner-content {
     margin-bottom: 7.5rem;
     margin-top: 7rem;
+}
+
+/* main filter and dropdowns *******************************/
+  .find-out__filter {
+    @include media(small) {
+      display: grid;
+      grid-template-columns: 1fr 70% 1fr;
+      padding-top: 9.35rem;
+    }
+
+    display: grid;
+    grid-template-columns: 1fr 41rem 1fr;
+    padding-top: 9.35rem;
+
+    &-content-type {
```

Comment:
Would these work as "elements" in BEM style? (i.e. `find-out__content-type`, `find-out__topic-area`)

Reply:
Since "content_type" and "topic_areaa" are under 'nested' under `.find-out__filter`, I treated them as modifiers of the `filter` element.  Hence `.find-out__filter-content-type` and `.find-out__filter-topic-area`.  Later on in the code, there are other references to `content-type` and `topic-area` which nest under other elements.  So that's why I ended up with this state. But the whole file could use a refactor from a bigger BEM perspective.  
Also, since our discovery this morning of the IE issue with grid, I am having to refactor this with Flex, since the autoprefixer gem does not seem to fix it, nor does an NPM autoprefixer that I tried, and directly editing the grid code with `-ms-` prefixes didn't work either.  So I am doing it over in CSS Flex.